### DISCUSSION TITLE
讨论个问题，大家使用table并分页时，会每次翻页都进行下总条数的检索吗？

### DISCUSSION THREAD

**Discussion by cnazev** [layui/layui#1560] [问答]
如果翻页过程中有了一些新数据插入，大家会按照新的数据更新吗？

**Comment by sentsim** [comment]
一般会，若每次翻页都是通过服务端返回的数据进行渲染，那么每次翻页得到的必定是当前页最新的数据

**Reply by cnazev** [reply]
多谢，一直不太确定该如何选择，使用新数据的话就是可能会有一点小混乱，比如刚才看了第二页，翻到第三页后发现有些数据是刚才看过的第二页里的。但也只是有这一点小瑕疵。使用旧数据的话就是程序里得用个条件限制住，比如时间字段或者ID之类。

**Comment by LSL1618** [comment]
这个一般取决于后端接口的查询语句排序规则，如果是旧数据排在前面，原先几页的数据就不会变，如果是新数据排在前面，那原先几页的数据就一定会变。

**Comment by elylon1101** [comment]
主要还是看数据量，如果数据量太大的项目直接不返回总条数，如果数据量小就实时查询总条数
